hcc_config/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
+if (HCC_RUNTIME_DEBUG)
+  add_compile_options(-g -O0)
+endif (HCC_RUNTIME_DEBUG)
+
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/../hcc_config)
+
+set(CMAKE_CXX_FLAGS "-std=c++11" )
+add_definitions(-DLIBCXX_HEADER=${LIBCXX_HEADER}) 
+
+add_executable(hcc-config hcc_config.cpp)
+
+
+add_custom_command(TARGET hcc-config POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E create_symlink ./hcc-config ./clamp-config
+  WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/bin
+)
+
+install(TARGETS hcc-config
+    RUNTIME DESTINATION bin
+)
+
+install(FILES ${PROJECT_BINARY_DIR}/bin/clamp-config DESTINATION bin)
```

Comment:
Yes because the symlink is relative path, here's the content of the install dir created by the debian package

scchan@d7e2ff423cf3:/opt/rocm/hcc-1.0/bin$ ls -la
total 36
drwxrwxr-x 2 root root  4096 Dec 20 06:36 .
drwxrwxr-x 6 root root  4096 Dec 20 06:36 ..
lrwxrwxrwx 1 root root    12 Dec 20 05:56 clamp-config -> ./hcc-config
lrwxrwxrwx 1 root root    21 Dec 20 05:56 clang -> ../compiler/bin/clang
lrwxrwxrwx 1 root root    23 Dec 20 05:56 clang++ -> ../compiler/bin/clang++
-rwxr-xr-x 1 root root  5309 Dec 16 16:27 extractkernel
lrwxrwxrwx 1 root root    19 Dec 20 05:56 hcc -> ../compiler/bin/hcc
-rwxr-xr-x 1 root root 20136 Dec 20 05:55 hcc-config
lrwxrwxrwx 1 root root    19 Dec 20 05:56 lld -> ../compiler/bin/lld
lrwxrwxrwx 1 root root    23 Dec 20 05:56 llvm-mc -> ../compiler/bin/llvm-mc
lrwxrwxrwx 1 root root    28 Dec 20 05:56 llvm-objdump -> ../compiler/bin/llvm-objdump